lib/plugins/inspec-parallel/lib/inspec-parallel/runner.rb

```diff
@@ -34,7 +34,7 @@ def run
           cleanup_child_processes
           sleep 0.1
         end
-        cleanup_empty_error_log_files
+        cleanup_empty_error_log_files unless Inspec.locally_windows?
```

Comment:
While InSpec Parallel exec now works on Windows, it generates spurious error logs regarding deleting empty log files. I don't think it is so important to delete empty log files that we should cause an error. I think - at least temporarily - on windows we should not delete empty error log files.